code/modules/library/skill_learning/skill_station.dm

```diff
@@ -104,8 +104,12 @@
 // Functions below do not validate occupant exists - should be handled outer wrappers.
 /// Start implanting.
 /obj/machinery/skill_station/proc/start_implanting()
-	if(!inserted_skillchip?.can_be_implanted(occupant))
+	var/mob/living/carbon/carbon_occupant = occupant
+
+	if(inserted_skillchip.has_mob_incompatibility(carbon_occupant))
+		stack_trace("Unusual error - [usr] attempted to start implanting of [inserted_skillchip] when the interface state should not have allowed it.")
```

Comment:
`stack_trace` is just `CRASH` without a return.
Use the latter if there is one.